DeAceroDLabs/marketplace-frontend
# [DMP-433] Updated API language

## What is done

[[DMP-433] Update API language](https://deacero.atlassian.net/browse/DMP-433)

## How is done

1. Changed the language of the API properties from spanish to english.
2. Changed the cases where the API is being used to avoid errors.
3. Modified the tests to match the new API properties names.

## How to update the Backend

1. Go to the location of your `mkt-place-back` location.
2. Stop the containers `docker compose down`
3. Remove the postgress container by running `docker rm mkt-place-back-postgres-1`
4. Remove the backend container by running `docker rm mkt-place-back-mkt-place-back-1`
5. Remove the images:
- List the running images `docker images -a`
- Remove each image `docker image rm <image id>`
6. Checkout to the master branch.
7. Pull the latest changes.
8. Run the backend project `docker compose up`

## How to test

1. Run the project via `npm start`.
2. Login to the app.
3. Check that everything works as usual on the categories section.
4. Check that everything works as usual on the Search page.
